memcached/memcached.go

```diff
@@ -122,15 +122,21 @@ func (c *CacheClient) AddItem(key string, value interface{}, expiration int32) (
 func (c *CacheClient) UpdateItem(key string, value interface{}, expiration int32, addIfNotExists bool) (bool, error) {
 	item, err := CreateMemCacheObject(key, value, expiration)
 	if err != nil {
-		return false, err
+		c.logger.LogError("Error occurred while updating item in cache.", err)
+		return false, nil
```

Comment:
Why remove the err?

Reply:
Because, it should not be a blocker, if update fails, it should be logged and false is returned as a status, that update did not happen.

Reply:
This will not be backwards compatible.

Reply:
As discussed.